services/121-service/src/payments/fsp-integration/commercial-bank-ethiopia/commercial-bank-ethiopia.module.ts

```diff
@@ -0,0 +1,47 @@
+import { HttpModule } from '@nestjs/axios';
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { WhatsappModule } from '../../../notifications/whatsapp/whatsapp.module';
+import { ProgramFspConfigurationEntity } from '../../../programs/fsp-configuration/program-fsp-configuration.entity';
+import { ProgramEntity } from '../../../programs/program.entity';
+import { RegistrationEntity } from '../../../registration/registration.entity';
+import { CustomHttpService } from '../../../shared/services/custom-http.service';
+import { UserEntity } from '../../../user/user.entity';
+import { UserModule } from '../../../user/user.module';
+import { SoapService } from '../../../utils/soap/soap.service';
+import { ImageCodeModule } from '../../imagecode/image-code.module';
+import { TransactionEntity } from '../../transactions/transaction.entity';
+import { TransactionsModule } from '../../transactions/transactions.module';
+import { CommercialBankEthiopiaApiService } from './commercial-bank-ethiopia.api.service';
+import { CommercialBankEthiopiaMockService } from './commercial-bank-ethiopia.mock';
+import { CommercialBankEthiopiaService } from './commercial-bank-ethiopia.service';
+
+@Module({
+  imports: [
+    HttpModule,
+    TypeOrmModule.forFeature([
+      RegistrationEntity,
+      TransactionEntity,
+      ProgramEntity,
+      ProgramFspConfigurationEntity,
+      UserEntity,
+    ]),
+    ImageCodeModule,
```

Comment:
Is this used somewhere? I can't find it in code. If not it's not used, please remove

Reply:
You are right and I have removed unused modules.